src/app/clusters/time-format-localization-server/time-format-localization-server.cpp

```diff
@@ -52,44 +52,47 @@ class TimeFormatLocalizationAttrAccess : public AttributeAccessInterface
     CHIP_ERROR ReadSupportedCalendarTypes(AttributeValueEncoder & aEncoder);
 };
 
+class AutoReleaseIterator
```

Comment:
This should perhaps (in a followup) move to some shared place next to where we define our common iterators....

Reply:
I think we should have a common iterator for our iterator things, however I was unsure how common that was. At least `SupportedCalendarTypes` seems highly specific so I would not move as-is but rather see about templating it.

Separate PR though.